<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plant Project</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="tabUmbr">
       <attribute name="title">
        <string>Umbralización</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="lblContraste">
          <property name="text">
           <string>Contraste:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cboUmbr"/>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="chkEjem">
          <property name="text">
           <string>Mostrar ejemplos</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="spacerUmb">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lblUmbr">
          <property name="text">
           <string>Tipo de Umbralización:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spnUmbContraste"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSegm">
       <attribute name="title">
        <string>Segmentación fondo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="2">
         <widget class="QCheckBox" name="chkAdaptativo">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Modelo adaptativo</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="spnSegmContraste"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lblSegmContraste">
          <property name="text">
           <string>Contraste:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Configuración modelo</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Espciado entre frames:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spnNum"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lblNum">
             <property name="text">
              <string>Número de frames:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spnSpace"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="2">
         <spacer name="spacerSegm">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabBgSub">
       <attribute name="title">
        <string>Sustracción de fondo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1">
         <widget class="QComboBox" name="cboAlg"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lblAlg">
          <property name="text">
           <string>Algoritmo de segmentación:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="spacerSub">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0" colspan="3">
        <widget class="QPushButton" name="pbEjecutar">
         <property name="text">
          <string>Ejecutar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblVideo">
         <property name="text">
          <string>Video:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pbExaminar">
         <property name="text">
          <string>Examinar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lneVideoPath"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
